ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -349,12 +350,12 @@ export const NetworkListMenu = ({ onClose }: { onClose: () => void }) => {
       await handleNonEvmNetworkChange(chainId);
     }
 
-    const [chainIdToTrack, currentChainIdToTrack] = isEvm
-      ? [
-          convertCaipToHexChainId(chainId),
-          convertCaipToHexChainId(currentChainId),
-        ]
-      : [chainId, currentChainId];
+    const chainIdToTrack = isEvm ? convertCaipToHexChainId(chainId) : chainId;
+    const currentChainIdToTrack = isScopeEqualToAny(currentChainId, [
+      'eip155:0',
+    ])
```

Comment:
You can use `isScopeEqual` for single scope instead

```suggestion
    const currentChainIdToTrack = isScopeEqual(currentChainId, EthScope.Eoa)
```

But can't we just `isEvm` here instead?

Reply:
How about this https://github.com/MetaMask/metamask-extension/pull/30614/commits/d057ace65e0f6e90fbd7fea08f045ead6b9ea4f1?